.github/actions/create-logo-pack/action.yml

```diff
@@ -0,0 +1,54 @@
+name: 'Create Logo Pack'
+description: 'Creates a ZIP archive of all brand logos'
+inputs:
+  logos-dir:
+    description: 'Directory containing logos'
+    required: false
+    default: 'application/public/brand/logo'
+  output-file:
+    description: 'Output ZIP file path'
+    required: false
+    default: 'application/public/brand/downloads/hoverkraft-logos.zip'
+outputs:
+  pack-created:
+    description: 'Whether the logo pack was successfully created'
+    value: ${{ steps.create-pack.outputs.success }}
+runs:
+  using: 'composite'
+  steps:
+    - name: Verify logos exist
+      id: verify
+      shell: bash
+      run: |
+        if [ -d "${{ inputs.logos-dir }}" ] && [ "$(find ${{ inputs.logos-dir }} -type f 2>/dev/null)" ]; then
+          echo "has-logos=true" >> $GITHUB_OUTPUT
+          echo "✓ Found logos in ${{ inputs.logos-dir }}"
+        else
+          echo "has-logos=false" >> $GITHUB_OUTPUT
+          echo "::warning::No logos found in ${{ inputs.logos-dir }}"
+        fi
+    
+    - name: Create logo pack using zip-release
+      id: create-pack
+      if: steps.verify.outputs.has-logos == 'true'
+      uses: TheDoctor0/zip-release@b57d897cb5d60cb78b51a507f63fa184cfe35554 # v0.7.6
+      with:
+        type: 'zip'
+        filename: '${{ inputs.output-file }}'
+        directory: '${{ inputs.logos-dir }}'
+        path: '.'
+    
+    - name: Verify ZIP created
+      if: steps.verify.outputs.has-logos == 'true'
+      shell: bash
+      run: |
+        if [ -f "${{ inputs.output-file }}" ]; then
+          echo "✓ Created logo pack: ${{ inputs.output-file }}"
+          echo "Logo pack contents:"
+          unzip -l "${{ inputs.output-file }}" || true
+          echo "success=true" >> $GITHUB_OUTPUT
+        else
+          echo "::error::Failed to create logo pack"
+          echo "success=false" >> $GITHUB_OUTPUT
+          exit 1
+        fi
```

Comment:
prefers using github action script step instead of run.

If there is something wrong use return core.setFailed